src/config/2021.json

```diff
@@ -528,6 +545,14 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/22671477?v=4&s=200",
       "github": "shantsis"
     },
+    "boosef": {
```

Comment:
FYI @boosef adding your contributor info. Open a new PR to make any changes.

Reply:
Looks good to me, thanks! :)